Brief-over-of-Heads-and-firmware-security/Heads-threat-model.md

```diff
@@ -0,0 +1,471 @@
+---
+layout: default
+title: Heads Threat model
+permalink: /Heads-threat-model/
+nav_order: 2
+parent: Brief overview of Heads and firmware security
+---
+
+The threat model that Heads proposes to address is very different from that of
+ Tails. Tails's goal is to allow users to to do computation on a machine in a
+ way that doesn't leave in trace on that system. This requires that the hardware
+ in the system is trusted, which unfortunately is not the case for many users.
+ Additionally many users need a way to keep state in a permanent way and don't
+ want to expose this state to random machines. Their machines might be subject
+ to physical attacks that might install untrusted firmware or other devices into
+ the system.
+
+* [LightEater malware seek GPG keys in Tails, Kallenberg and Kovah, 2015](https://www.theregister.com/2015/03/19/cansecwest_talk_bioses_hack/)
+* [Thunderstrike, Hudson 2014](https://trmm.net/Thunderstrike)
+
+For these reasons, Tails is not sufficient for many users who want a laptop that
+ they can travel with and want to have some assurances that most adversaries
+ won't be able to modify the hardware underneath them.
+
+Complicating this goal is that modern x86 hardware is full of modifiable state
+ [State considered harmful, Rutkowska 2015](https://blog.invisiblethings.org/2015/12/23/state_harmful.html)
```

Comment:
Previous refs for state considered harmful is blog post here, which links to pdf. Might be better to just reuse.